### ISSUE TITLE
Border module includes invalid outline properties

### ISSUE THREAD

**Issue by nsaunders** [sebastiaanvisser/clay#223]
[MDN's `border-style`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-style#constituent_properties) documentation, as an example, shows "constituent properties" that allow a unique border to be applied to each side of an element. However, this does not seem to be the case for outlines: Notice, for example, how [`outline-style`](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-style) has no constituent properties listed. Indeed, my own testing suggests that outline properties can't be assigned to each side of an element independently: For example, `outline-top-color` yields an "unsupported property" warning in Safari:

<img width="286" alt="image" src="https://user-images.githubusercontent.com/2517229/153413077-f6826ac3-62be-438c-b8c3-1f149e5f4ddb.png">

**Comment by nsaunders** [comment]
@turion that is correct. These are the only outline functions that produce valid CSS:

* `outline`
* `outlineColor`
* `outlineStyle`
* `outlineWidth`
* `outlineOffset`

So these should be removed:

* `outlineTop`
* `outlineRight`
* `outlineBottom`
* `outlineLeft`
* `outlineColor4`
* `outlineTopColor`
* `outlineRightColor`
* `outlineBottomColor`
* `outlineLeftColor`
* `outlineStyle4`
* `outlineTopStyle`
* `outlineRightStyle`
* `outlineBottomStyle`
* `outlineLeftStyle`
* `outlineWidth4`
* `outlineTopWidth`
* `outlineRightWidth`
* `outlineBottomWidth`
* `outlineLeftWidth`

**Comment by turion** [comment]
PRs welcome to this issue :)
